lib/config/typescript-prettier.js

```diff
@@ -1,19 +1,4 @@
 module.exports = {
+  // DEPRECATED: plugin:shopify/prettier is sufficient now
   extends: ['plugin:shopify/prettier'],
-
-  rules: {
-    // rules to disable to prefer prettier
-    'typescript/member-delimiter-style': 'off',
```

Comment:
Disabling this rule should still be present (but in an overrides section prettier) as we want to defer to prettier rather than risk configuration of conflicting with prettier's output. (Note the default value when enabling this rule matches prettier, but people may have changed it to something else)

Reply:
would leaving this rule in our config not override what is defined in a `.prettierrc` file?

Reply:
The typescript ruleset [configures "typescript/member-delimiter-style"](https://github.com/Shopify/eslint-plugin-shopify/blob/4fe4585365b1a91f9bfedfd3beccf15ad68a526d/lib/config/rules/typescript.js#L87). When using prettier, we want to disable that eslint rule, and instead trust whatever prettier outputs.